extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetTools.java

```diff
@@ -604,15 +618,26 @@ private static ParquetSchemaReader.ColumnDefinitionConsumer makeSchemaReaderCons
      */
     public static ParquetFileReader getParquetFileReader(@NotNull final File parquetFile) {
         try {
-            return new ParquetFileReader(
-                    parquetFile.getAbsolutePath(),
-                    new CachedChannelProvider(
-                            new TrackedSeekableChannelsProvider(TrackedFileHandleFactory.getInstance()), 1 << 7));
+            return getParquetFileReaderIo(parquetFile);
         } catch (IOException e) {
             throw new TableDataException("Failed to create Parquet file reader: " + parquetFile, e);
         }
     }
 
+    /**
+     * Make a {@link ParquetFileReader} for the supplied {@link File}.
+     *
+     * @param parquetFile The {@link File} to read
+     * @return The new {@link ParquetFileReader}
+     * @throws IOException if an IO exception occurs
+     */
+    public static ParquetFileReader getParquetFileReaderIo(@NotNull File parquetFile) throws IOException {
```

Comment:
I want to paint this bike shed a different color, I just don't have a great suggestion. `getParquetFileReaderChecked`?

Reply:
Sure.